.github/workflows/cicd.yaml

```diff
@@ -344,3 +344,89 @@ jobs:
         run: |
           ls
           twine upload openmldb-*.whl
+
+  build-and-publish-docker:
+
+    runs-on: ubuntu-latest
+    needs: [ "release" ]
```

Comment:
Can't run docker compose if somebody send a pr

Reply:
Run over a tag push might hard to maintain, two level dependency chain. 

There are just two condition to run the job:
1. Somebody touch the docker directory, runs test on pull request
2. Watch to publish a new version. And i think publish on manual dispatch event with the version parameter might cleaner.

Anyway, if those docker was in a separate repo, things might more clear